client/src/modules/dashboard/Events/components/EventForm.tsx

```diff
@@ -283,7 +288,7 @@ const EventForm: React.FC<EventFormProps> = (props) => {
             <h1>Error loading venues</h1>
           ) : (
             isPhysical(getValues('venue_type')) && (
-              <FormControl isRequired>
+              <FormControl marginBlock={'1em'}>
```

Comment:
```suggestion
              <FormControl isRequired marginBlock={'1em'}>
```
Same principle as above - this one is required.